test/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+CompetingClocks = "5bb9b785-358c-4fee-af0f-b94a146244a8"
```

Comment:
This may be a red herring. When the Test is usually run, it adds the project to the test dependencies for you. Don't worry about this, though. I'll update the tests to use TestItems.